keyboards/dztech/bocc/bocc.h

```diff
@@ -21,8 +21,8 @@
 	K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012, K013, K014, K414, K015, K016, \
 	K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K112, K113, K114, K115, K116,       \
 	K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213, K214,       K216,       \
-	K300, K301, K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, K312, K313,       K315, K316,       \
-	K400, K401, K402, K403, K404, K405,       K407,                   K411, K412, K413,       K415, K416        \
+	K300, K405, K301, K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, K312, K313, K315, K316,       \
```

Comment:
You moved the keycode in the keymaps to the fourth position on this row, but the identifier in the macro to the second.

```suggestion
	K300, K301, K302, K405, K303, K304, K305, K306, K307, K308, K309, K310, K311, K312, K313, K315, K316,       \
```